rcljava/src/main/cpp/org_ros2_rcljava_RCLJava.cpp

```diff
@@ -58,20 +59,73 @@ Java_org_ros2_rcljava_RCLJava_nativeCreateNodeHandle(
 
   rcl_context_t * context = reinterpret_cast<rcl_context_t *>(context_handle);
 
+  // TODO(clalancette): we could probably make this faster by just doing these
+  // method lookups during some initialization time.  But we'd have to add a lot
+  // more infrastructure for that, and we don't expect to call
+  // 'nativeCreateNodeHandle' that often, so I think this is OK for now.
+  jclass java_util_ArrayList =
+    static_cast<jclass>(env->NewGlobalRef(env->FindClass("java/util/ArrayList")));
```

Comment:
This is creating a global reference each time `Java_org_ros2_rcljava_RCLJava_nativeCreateNodeHandle`, and never deleting it with `DeleteGlobalRef`. Thus, it's leaking.

Creating a `GlobalRef` is not needed, you can just use the value returned by `FindClass`.

Reply:
I didn't realize that, thanks.  Will clean up.

Reply:
Fixed in e693055